docs/screencasts.json

```diff
@@ -40,5 +47,20 @@
     "title": "Firebase Cloud Firestore Overview",
     "description": "Learn how to setup Firebase Cloud Firestore for your React Native app, a NoSQL real-time database for storing your application data.",
     "tags": ["firestore"]
+  },
+
+  {
+    "slug": "dynamic-links-overview",
+    "vid": "yXuIw1PznvY",
+    "title": "Firebase Dynamic Links Overview",
+    "description": "Learn how to setup Dynamic Links from Google's Firebase.",
+    "tags": ["dynamic-links"]
+  },
+  {
+    "slug": "enabling-multidex-for-android",
+    "vid": "4oVGEgHR3KE",
+    "title": "Enabling Multidex fix for Android",
+    "description": "As more native dependencies are added to your project, in particular firebase, it may bump you over the 64k method. Learn how to update the limit on the Android build system.",
```

Comment:
```diff
- in particular firebase
+ in particular Cloud Firestore
```

👀

Reply:
oh poop - totally correct on that thinko will fix @Salakar

Reply:
https://github.com/invertase/react-native-firebase/commit/63b822cd404892a61403c1c4c10d91a8ec4f13cc